kaggle_environments/agent.py

```diff
@@ -68,104 +95,37 @@ def url_agent(o, c, r, i):
         raw = read_file(raw, raw)
 
     # Attempt to execute the last callable or just return the string.
-    try:
-        local = get_exec(raw)
-        callables = [v for v in local.values() if callable(v)]
-        if len(callables) > 0:
-            return callables[-1]
-        raise "Nope"
-    except:
-        return lambda: raw
-
-
-def run_agent(agent, message):
-    if message.state != None and message:
-        args = [
-            structify(message.state["observation"]),
-            structify(message.configuration),
-            message.state["reward"],
-            structify(message.state["info"])
-        ][:agent.__code__.co_argcount]
-        try:
-            message.action = agent(*args)
-        except Exception as e:
-            message.action = e
-        message.state = None
-
-
-def runner(raw, message, environment, debug=False):
```

Comment:
All of this stuff was previously used to create a process wrapper and polling loop but none of it is used so removing.